nautobot_chatops/utils.py

```diff
@@ -14,6 +14,63 @@
 logger = logging.getLogger(__name__)
 
 
+try:
+    from nautobot.core.celery import nautobot_task
+
+    @nautobot_task
+    def celery_worker_task(command, subcommand, params, dispatcher_module, dispatcher_name, context):
+        """Task executed by Celery worker.
+
+        Celery cannot serialize/deserialize objects, instead of passing the
+        function, registry or dispatcher_class, we will have to look them up.
+        """
+        # import done here instead of up top to prevent circular imports.
+        # Disabling cyclic-import as this does not affect the worker.
+        # pylint: disable=import-outside-toplevel,cyclic-import
+        from nautobot_chatops.workers import get_commands_registry
+
+        registry = get_commands_registry()
+        function = registry[command]["function"]
+        # Get dispatcher class from module
+        dispatcher_class = getattr(sys.modules[dispatcher_module], dispatcher_name)
+
+        return function(subcommand, params=params, dispatcher_class=dispatcher_class, context=context)
+
+    def enqueue_task(command, subcommand, params, dispatcher_class, context, function=None):
+        """Enqueue task with Celery worker."""
+        # Since celery can't deserialize class, we will discard function and
+        # send the command instead.
+        _ = function
```

Comment:
I assume the assignment to `_` is just to avoid a `pylint:disable=unused-argument` comment? Personally I'd prefer the comment approach versus having actual do-nothing code.

Reply:
Yeah, it was trying to avoid that.  I will change.